gryphon/lib/exchange/binance_je.py

```diff
@@ -0,0 +1,188 @@
+"""
+https://github.com/binance-jersey/binance-official-api-docs
+"""
+
+# -*- coding: utf-8 -*-
```

Comment:
I believe this line has to be the first line in the file.

Reply:
I thought so too - but I copied it from one of the other implementations! I'll move it.